.github/workflows/ci-sql-cli.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
       fail-fast: false
       matrix:
         python: [ 3.8, 3.9 ]
-        airflow: [ 2.2.5, 2.3.4, 2.4.2 ]
+        airflow: [ 2.2.5, 2.3.4, 2.4.2, 2.5 ]
```

Comment:
```suggestion
        airflow: [ 2.2.5, 2.3.4, 2.4.2, 2.5.0 ]
```

Reply:
@feluelle I believe we should be only adding the micro release if we're using constraints, WDYT?

Reply:
this would allow us to test with the latest micro releases - when possible - without us having any effort

Reply:
I am totally for it. Could you change this for 2.3 and 2.4 here as well?

Reply:
We are not using constraints file though for >2.2, or are you talking about different constraint?

Reply:
It's the same constraint, @kaxil !
I attempted removing constraints of 2.3.4 and 2.4.2 - it worked for 2.4, but in the case of 2.3, without constraints, it raised an issue due to an outdated click version:
```
 ImportError while loading conftest '/home/runner/work/astro-sdk/astro-sdk/sql-cli/conftest.py'.
conftest.py:12: in <module>
    from sql_cli.dag_generator import Workflow
sql_cli/dag_generator.py:7: in <module>
    from black import FileMode, format_str
src/black/__init__.py:31: in <module>
    ???
E   ImportError: cannot import name 'ParameterSource' from 'click.core' (/home/runner/work/astro-sdk/astro-sdk/sql-cli/.nox/test-3-9-airflow-2-3/lib/python3.9/site-packages/click/core.py)
```

E.g.:
https://github.com/astronomer/astro-sdk/actions/runs/3630597057

Reply:
It fails because of recently added `black`.

Instead of:
```
black = { version = "*", allow-prereleases = true }
```

we should have:
```
black = ">=22.10.0"
```

Reply:
Ah it was using the [latest version](https://github.com/astronomer/astro-sdk/actions/runs/3630597057/jobs/6124794996#step:7:496) already.